services/common/src/components/reports/ReportDetailsForm.tsx

```diff
@@ -243,6 +249,39 @@ const ReportDetailsForm: FC<ReportDetailsFormProps> = ({
         initialValues={initialValues}
       >
         <Row gutter={[16, 8]}>
+          {system === SystemFlagEnum.core && (
+            <Col span={24}>
+              <Typography.Title level={3} id="regulatory-authority">
+                Regulatory Authority
+              </Typography.Title>
+
+              <Field
+                name="report_for"
+                id="report_for"
+                required
+                disabled={true}
+                props={{
+                  isVertical: true,
+                }}
+                label="Who is the report for?"
+                component={RenderRadioButtons}
+                validate={[requiredRadioButton]}
+                customOptions={[
+                  { label: "Chief Permitting Officer", value: "CPO" },
+                  { label: "Chief Inspector of Mines", value: "CIM" },
```

Comment:
hindsight- we should have used CHI and CHP. 😅 
Backend: we can query by those values,
![image](https://github.com/bcgov/mds/assets/102187683/0cf57370-8793-424d-8ce0-427ee37a2e65)
Frontend: they're already in Core Static Content (I was looking to see if there was an enum somewhere)
![image](https://github.com/bcgov/mds/assets/102187683/f8a021f1-1169-48a9-a171-597c8be5f3d0)

Reply:
We can confirm with Rebecca. From the last work I did on the data side of this, there are only 3 possible values (CIM, CPO, Both) as specified in the spreadsheet of the a/c (MDS-5472). It now makes sense that the description of those codes are the same as some of the EMLI contact types. I only wonder what the code should be when it is Both.

Reply:
Just leaving a note here, but it was decided by the team to leave it as it is for now.